### DISCUSSION TITLE
Is it possible to parse the field json?

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#6292] [Help]
There is a Project model, it has a props field in json format
`
{
    "area": "170",
    "address": "Московская область, Россия",
    "typeHouse": "Деревянный брус 150 х 150 под обшивку блокхаусом",
    "typeHome": "Дом",
    "year": "2017"
}
`
is there any way to make each property a separate field in the admin panel?

**Comment by danharrin** [comment]
Yes, `TextInput::make('props.area')`
